itests/kit/docker/docker.go

```diff
@@ -0,0 +1,119 @@
+package docker
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// Container tracks information about the docker container started for tests.
+type Container struct {
+	ID   string
+	Host string // IP:Port
+}
+
+// StartTendermintContainer starts a Tendermint container for running tests.
+func StartTendermintContainer() (*Container, error) {
+	dir, err := os.Getwd()
+	if err != nil {
+		return nil, fmt.Errorf("could not get current dir: %w", err)
+	}
+
+	var out, stdErrOut bytes.Buffer
+
+	// Init Tendermint
+
+	cmd := exec.Command(
+		"docker", "run", "--rm",
+		"-v", dir+"/../testdata/tendermint:/tendermint",
+		"--entrypoint", "/usr/bin/tendermint",
+		"tendermint/tendermint:v0.35.1",
+		"init", "validator", "--key", "secp256k1")
+	cmd.Stdout = &out
+	cmd.Stderr = &stdErrOut
```

Comment:
I don't think we should worry about this now, but this may fail for people that require sudo privileges to run docker :)

Reply:
ok